docs/source/resources/graphql-glossary.md

```diff
@@ -0,0 +1,353 @@
+---
+title: GraphQL Glossary
+description: A comprehensive list of important GraphQL words and acronyms
+---
+
+
+When you start diving into the GraphQL ecosystem, you'll probably encounter some unfamiliar terms and phrases along the way. To help you on your journey, we've defined some of the most common GraphQL vocabulary here in this handy cheat sheet.
+
+
+<h2 id="Apollo">Apollo</h2>
+<p>An open-source implementation of GraphQL that helps you manage data between the cloud and your UI. The Apollo platform is pluggable into your existing architecture and features production-ready tooling across the stack ([Server](https://www.apollographql.com/docs/apollo-server/getting-started.html), [Client](https://www.apollographql.com/docs/react/), and [Engine](https://www.apollographql.com/docs/engine/)).</p>
+
+<h2 id="automatic-persisted-queries">Automatic Persisted Queries (APQ) </h2>
+<p> A technique for improving GraphQL network performance with zero build-time configuration by reducing request size over the wire. A smaller signature reduces bandwidth utilization and speeds up client loading times. Apollo Server allows implementation of [Automatic Persisted Queries (APQ)](https://www.apollographql.com/docs/guides/performance.html#automatic-persisted-queries). </p>
+
+<h2 id="argument">Argument</h2>
+<p>A set of key-value pairs attached to a specific field. Arguments can be literal values or variables.</p>
+
+```js
+{
+  human(id: "200") {
+    weight(unit: "pounds")
+    height
+  }
+}
+```
+
+<h2 id="alias">Alias</h2>
+<p>An alternative name given to the result of a field to avoid conflicts during data fetching.</p>
+
+```js
+{
+  admin: users(role: admin) {
+    id
+    firstname
+    lastname
+  }
+  managers: users(role: manager) {
+    id
+    firstname
+    lastname
+  }
+}
+```
+
+`admin` and `managers` are aliases in the example query above.
+
+<h2 id="data-source">Data Source</h2>
```

Comment:
Can you please expand upon this definition a little more and link to the data source docs?